src/XTabs/_XTabHeader.razor

```diff
@@ -110,4 +110,10 @@
         _dropClass = string.Empty;
         return;
     }
+
+    private void CloseTabOnMouseWheel(XTab xTab, MouseEventArgs e)
+    {
+        if (e.Button != 1) return;
+        OnCloseTab.InvokeAsync(xTab);
```

Comment:
Async call. Can the method be turned into async?

Reply:
Yeah, no problem!